Sources/Receipt/ReceiptDetailTableViewCell.swift

```diff
@@ -28,6 +28,13 @@ final class ReceiptDetailTableViewCell: UITableViewCell {
         super.init(coder: aDecoder)
     }
 
+    func configure(configuration: (title: String, value: String)) {
+        textLabel?.text = configuration.title
+        textLabel?.accessibilityIdentifier = configuration.title
+        detailTextLabel?.text = configuration.value
+        detailTextLabel?.accessibilityIdentifier = configuration.value
```

Comment:
let's change it to "\(configuration.title)_value" instead of configuration.value

Reply:
fixed